src/main/kotlin/com/cognifide/gradle/aem/instance/LocalInstance.kt

```diff
@@ -220,14 +220,19 @@ class LocalInstance private constructor(aem: AemExtension) : AbstractInstance(ae
         }
 
         // Ensure that 'logs' directory exists
-        GFileUtils.mkdirs(File(staticDir, "logs"))
+        GFileUtils.mkdirs(File(quickstartDir, "logs"))
     }
 
-    private fun extractStaticFiles() {
-        aem.logger.info("Extracting static files from JAR '$jar' to directory '$staticDir'")
+    private fun unpackFiles() {
+        aem.logger.info("Unpacking quickstart from JAR '$jar' to directory '$quickstartDir'")
 
-        aem.progress(FileOperations.zipCount(jar, JAR_STATIC_FILES_PATH)) {
-            FileOperations.zipUnpack(jar, staticDir, JAR_STATIC_FILES_PATH) { increment("Extracting file '$it'") }
+        aem.progressIndicator {
+            message = "Unpacking quickstart JAR: ${jar.name}, size: ${Formats.size(jar)}"
+            aem.project.javaexec { spec ->
```

Comment:
thx @toniedzwiedz for sharing knowledge :)

Reply:
Glad to have helped :)